src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/DescendantToken/context/defaultContextConfig.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+          "https://checkstyle.org/dtds/configuration_1_3.dtd">
+<module name="Checker">
+    <module name="TreeWalker">
+        <module name="DescendantToken">
+            <property name="tokens" value="EQUAL"/>
+            <property name="limitedTokens" value="LITERAL_NULL"/>
+            <property name="maximumNumber" value="0"/>
+            <property name="maximumDepth" value="1"/>
+        </module>
+
+        <module name="DescendantToken">
```

Comment:
please explain what for this complexity with few configs of Check ? please do single Check config block in XML.
lets keep it simple. If you want ... please create separate test folder to test Check in different configuration.

Reply:
done